draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -520,6 +528,83 @@ Author:  Paul Hoffman, paul.hoffman@icann.org
 Change controller:  IESG
 ~~~~~
 
+# Privacy Considerations {#PrivacyConsiderations}
+
+{{RFC7626}} discusses DNS Privacy Considerations in both "On the
+wire" (Section 2.4), and "In the server" (Section 2.5) contexts. This
+is also a useful framing for DoH's privacy considerations.
+
+## On The Wire {#OnTheWire}
+
+DoH encrypts DNS traffic and requires authentication of the
+server. This mitigates both passive surveillance {{RFC7258}} and
+active attacks attempting to divert DNS traffic to rogue servers
+({{RFC7626}} Section 2.5.1). DNS over TLS {{RFC7858}} provides
+similar protections, while direct UDP and TCP based transports are
+vulnerable to this class of information leak.
+
+Additionally, the use of the HTTPS default port 443 and the ability to
+mix DoH traffic with other HTTPS traffic on the same connection can
+deter on-path devices from interfering with DNS operations and make
+DNS traffic analysis more difficult.
+
+## In The Server {#InTheServer}
+
+A DoH implementation is built on IP, TCP, TLS, and HTTP. Each layer
+contains one or more common features that can be used to correlate
+different queries to the same identity. DNS transports will generally
+carry the same privacy properties of the layers used to implement
+them. For example, the properties of IP, TCP, and TLS apply to DNS
+over TLS implementations.
```

Comment:
Didn't we also specify DNS over DTLS?

Reply:
I'm not trying to enumerate every privacy consideration of every dns protocol. The point that I believe the WG wants to convey is primarily that DoH is a bit different from DoT, and I'm trying to be illustrative about the similarities and differences.

also it looks like 8094 which carries this lovely note: This DTLS solution was considered by the DPRIVE working group as a potential option to use in case that the TLS based approach specified in RFC7858 is shown to have detrimental deployment issues. At the time of writing, it was expected that RFC7858 will be deployed, and so this specification is primarily intended as a backup and has therefore been designated as experimental. This solution should not be deployed in the wild while in this experimental state as an RFC, however experimentation is encouraged.

Reply:
OK